lib/config/typescript.js

```diff
@@ -1,8 +1,6 @@
 const merge = require('merge');
 
 module.exports = {
-  extends: 'plugin:shopify/esnext',
-
   plugins: ['@typescript-eslint'],
 
   settings: {
```

Comment:
I think we are better off ditching the TS resolver too, anything that can be done in there can also be done with a bit more manual (but, generally, more correct) configuration of the default node resolver.

Reply:
(comment for `import/resolver` setting below)

Reply:
I think we leave the resolver change off for now. There is already a lot going on with this release.

Reply:
Please consider it. I know they will need to redo the paths logic, but a) the paths logic breaks quilt, which necessitated the gross workaround in that PR, and related b) the way the TS resolver works is that anything with `paths` is considered an internal module (since it resolves all things to source), where we typically want it considered differently (internal is fine for the "include `./app` in the list of folders to resolve from" situation, but treating them as external is preferable for the "include `./packages` in the list of folders to resolve from " situation). Also, we will probably slowly move away from paths, at least for cases where TS project references are more appropriate.

Reply:
@lemonmade sounds good, will remove the TS resolver in this PR 👍